docs/reference/migration/migrate_8_0/http.asciidoc

```diff
@@ -8,16 +8,31 @@
 
 // end::notable-breaking-changes[]
 
-[float]
-==== Removal of old HTTP settings
-
+.The `http.tcp_no_delay` setting has been replaced by `http.tcp.no_delay`.
+[%collapsible]
+====
+*Details* +
 The `http.tcp_no_delay` setting was deprecated in 7.x and has been removed in 8.0. It has been replaced by
 `http.tcp.no_delay`.
 
-[float]
-==== Changes to Encoding Plus Signs in URLs
+*Impact* +
+Use the `http.tcp.no_delay` setting. Discontinue use of the `http.tcp_no_delay`
+setting. Specifying the `http.tcp_no_delay` setting in `elasticsearch.yml` will
+result in an error on startup.
+====
 
+.The `es.rest.url_plus_as_space` system property has been removed.
+[%collapsible]
+====
+*Details* +
 Starting in version 7.4, a `+` in a URL will be encoded as `%2B` by all REST API functionality. Prior versions handled a `+` as a single space.
 If your application requires handling `+` as a single space you can return to the old behaviour by setting the system property
 `es.rest.url_plus_as_space` to `true`. Note that this behaviour is deprecated and setting this system property to `true` will cease
```

Comment:
This seems contradictory with the impact statement.

Reply:
Yeah. The following sentence makes it clear(er), and I did test to confirm that this property does get rejected. I reworded the original sentence with 001b32817f1d7a505bd2baa4b5c00cb4b1e03cd1.